test/shelf.test.js

```diff
@@ -34,8 +34,24 @@ describe("Generate usecase self-documentation", () => {
 
     })
 
+    before(function () {
+      env = process.env
+      process.env = { HERBS_SHELF_RENDER : "false" }
```

Comment:
We can improve the test scenarios for: HERBS_SHELF_RENDER = "false", HERBS_SHELF_RENDER="true", HERBS_SHELF_RENDER=true, HERBS_SHELF_RENDER=false, HERBS_SHELF_RENDER=0, HERBS_SHELF_RENDER=1